teachertool/src/components/styling/EvalResultDisplay.module.scss

```
@@ -102,6 +102,49 @@
         align-self: flex-end;
     }
 
+    .result-error {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: flex-start;
+        position: relative;
+        width: 100%;
+        padding: 0.5rem;
+        border: 1px solid var(--pxt-error);
+        border-radius: 1rem;
+        background-color: var(--pxt-error-background);
+        gap: 0.5rem;
+
+        .error-info-container {
+            display: flex;
+            flex-direction: column;
+            align-items: flex-start;
+            justify-content: space-between;
```

very minor suggestion. clip text to the error box when layout is very narrow:
```suggestion
            justify-content: space-between;
            overflow: hidden;
```